spec/reflexes/products_reflex_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "reflex_helper"
+
+describe ProductsReflex, type: :reflex do
+  let(:current_user) { create(:admin_user) } # todo: set up an enterprise user to test permissions
+  let(:context) {
+    { url: admin_products_v3_index_url, connection: { current_user: } }
+  }
+
+  before do
+    # activate feature toggle admin_style_v3 to use new admin interface
+    Flipper.enable(:admin_style_v3)
+  end
+
+  describe 'fetch' do
+    subject{ build_reflex(method_name: :fetch, **context) }
+
+    describe "sorting" do
+      let!(:product_z) { create(:simple_product, name: "Zucchini") }
+      # let!(:product_b) { create(:simple_product, name: "bananas") } # Fails on macOS
```

Comment:
what was the order on macOS?

Reply:
I believe it was ordering based purely on character code, so the capital letters first, then lowercase. Eg:

```
Apples    # char 65
Zucchini  # char 90 
bananas   # char 97
```

(See more in https://dba.stackexchange.com/questions/106964/why-is-my-postgresql-order-by-case-insensitive)

Reply:
Hm, I was wondering if this would impact production but we run Ubuntu and that seems to order correctly. So we are good.

Reply:
As long as we never use a BSD-based server I think we are safe!